api_tests/preprints/filters/test_filters.py

```diff
@@ -0,0 +1,113 @@
+# -*- coding: utf-8 -*-
+from nose.tools import *  # flake8: noqa
+import pytest
+from mock import MagicMock
+
+class PreprintsListFilteringMixin(object):
+
+    def setUp(self):
+        super(PreprintsListFilteringMixin, self).setUp()
+
+        self._setUp()
```

Comment:
Replied in wrong place, adding back here: The setup for the preprint_provider tests is different since all nodes must be the same provider, which is also why one of the tests is stubbed to pass.